docs/content/formulations/inertialParticles/Equations.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: default
+title: Governing Equations
```

Comment:
I don't think this will work without the parent files.  Did you add those?  Were you able to test it on your machine?

Reply:
Yes those are added to the following lines. No I did not test it on my machine. How can I do that? thanks.

Reply:
You can build and host locally following these directions https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/testing-your-github-pages-site-locally-with-jekyll

Reply:
Thanks, I am reading it now.